forge/test/mlir/llama/test_llama_inference.py

```diff
@@ -126,3 +127,141 @@ def test_llama_inference_cache_cpu():
     # Generated text
     generated_text = tokenizer.decode(generated_tokens[0], skip_special_tokens=True)
     print(generated_text)
+
+
+@pytest.mark.xfail()
+def test_llama_inference_no_cache_tt():
+
+    # Load Llama 3B model and tokenizer
+    model_path = "openlm-research/open_llama_3b"
```

Comment:
Please check few comments on this PR as well:
- https://github.com/tenstorrent/tt-forge-fe/pull/372

More precisely, maybe we can unify this `load_model` function to be a bit more generic :)) 

@dgolubovicTT please provide your insights as well based on Prefil on TT comments :))

Reply:
I would suggest to edit load_model method to return tokenizer too! so that. I will do that within https://github.com/tenstorrent/tt-forge-fe/pull/372.

Reply:
Replicated the change from PR https://github.com/tenstorrent/tt-forge-fe/pull/372 for load_model with minor change in the function name from `load_model` to `load_model_and_tokenizer`